src/SkirtBrim.cpp

```diff
@@ -370,16 +315,11 @@ Polygons SkirtBrim::getFirstLayerOutline(const int extruder_nr /* = -1 */)
 
         for (int i_layer = layer_nr; i_layer < skirt_height; ++i_layer)
         {
-            for (const auto& extruder : Application::getInstance().current_slice_->scene.extruders)
-            {
-                first_layer_outline
-                    = first_layer_outline.unionPolygons(storage_.getLayerOutlines(i_layer, include_support, include_prime_tower, external_only, extruder.extruder_nr_));
-            }
-        }
-
-        if (has_prime_tower)
-        {
-            first_layer_outline = first_layer_outline.unionPolygons(storage_.primeTower.getGroundPoly());
+            first_layer_outline = first_layer_outline.unionPolygons(storage_.getLayerOutlines(
+                i_layer,
+                /*include_support*/ true,
+                /*include_prime_tower*/ true,
+                true));
```

Comment:
Intention is pretty readable, but for consistency with the rest of the code base i would write something like
```suggestion
            constexpr include_support = true;
            constexpr include_prime_tower = true;
            first_layer_outline = first_layer_outline.unionPolygons(storage_.getLayerOutlines(
                i_layer,
                include_support,
                include_prime_tower,
                true));
```

Reply:
Indeed, I have a personal preference for this syntax, but consistency takes precedence :smiley: